src/lsp/cobol_lsp/lsp_diagnostics.ml

```diff
@@ -72,18 +72,20 @@ let publish diagnostics : unit =
 
 (* --- *)
 
-let as_notification ?(log = false) diag =
-  let type_ = match DIAG.severity diag with
-    | Hint | Note -> Lsp.Types.MessageType.Log
-    | Info -> Info
-    | Warn -> Warning
-    | Error -> Error
-  in
-  let message =
-    Pretty.(to_string "%t@[<h>%a@]" blast_margin DIAG.pp_msg diag) in
-  if log then
-    let params = Lsp.Types.LogMessageParams.create ~message ~type_ in
-    Lsp.Server_notification.LogMessage params
-  else
-    let params = Lsp.Types.ShowMessageParams.create ~message ~type_ in
-    Lsp.Server_notification.ShowMessage params
+(* let as_notification ?(log = false) diag = *)
```

Comment:
Yes indeed. I commented that as I noticed I was not used anymore, but it could still be useful later: for now the LSP does not send much notifications *per se*, but some more could be useful.